docs/source/contents/4_draft_mapping.rst

```diff
@@ -1784,6 +1784,280 @@ subject
 genre
 =====
 
++-----------------+--------------------+-------+
+| Predicate       | Value Type         | Range |
++-----------------+--------------------+-------+
+| dcterms:type    | URI/String Literal | N/A   |
++-----------------+--------------------+-------+
+| dcterms:subject | URI/String Literal | N/A   |
++-----------------+--------------------+-------+
+| edm:hasType     | URI/String Literal | N/A   |
++-----------------+--------------------+-------+
+
+genre[not(@*)]
+--------------
+
+Use Case
+^^^^^^^^
+`genre`, without any attributes, has been used as a catch-all descriptive element that may or may not hold values from a controlled vocabulary, and that may or may not provide appropriate descriptive information about the resource.
+
+Justification
+^^^^^^^^^^^^^
+For values *outside* of the following table, we selected the `edm:hasType` property as it aligns well with the possible overlap between `mods:genre` and `mods:physicalDescription/form`. To help prevent duplicating string literals and URIs, the following table suggests a mapping for a limited subset of the possible values in `genre[not(@*)]`.
+
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+| (//genre[not(@*] | //genre[@authority='dct']) | RDF Predicate | URI                                              | dcterms text value |
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+| cartographic                                  | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/car> | Cartographic       |
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+| image                                         | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/img> | Image              |
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+| notated music                                 | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/not> | Notated music      |
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+| still image                                   | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/img> | Still image        |
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+| text                                          | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/txt> | Text               |
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+
+XPath
+^^^^^
+:code:`genre[not(@*)]`
```

Comment:
@CanOfBees - the table looks good. For your Xpath I wondered if we needed to be more specific. You've sort of combined two categories into one. Thinking about this Xpath being machine actionable, how would we distinguish between the genre values that go into edm:hasType and those in dcterms:type? You've made this explicit in your Decision and Justification text, but the Xpath encompasses both. I'm wondering if splitting this into two cases where in one you say the Xpath is genre[string()="text"] etc. and in the other the Xpath is genre[not(string()="text")] would clarify this.

Reply:
@mlhale7 I tried to take a stab at merging the two sections, as well as updating the table and xpaths. I don't know if I'm happy with the results. :man_shrugging: Please let me know what you think about the changes - do they help clarify at all, or have I just made things worse?

Thanks!